scripts/rv_expression_association/get_rv_outliers.py

```diff
@@ -0,0 +1,168 @@
+import os
+import sys
+import click
+import logging
+import hail as hl
+import pandas as pd
+from cpg_utils.hail_batch import output_path
+
```

Comment:
```suggestion
from cloudpathlib import AnyPath

```

Reply:
AnyPath means that you can check for/open any type of path. `os.path` won't work for cloud paths

Reply:
ok, thank you!